Clicker/ViewControllers/CardController.swift

```diff
@@ -290,7 +290,9 @@ class CardController: UIViewController {
         presenter.dismissOnSwipeDirection = .bottom
         
         let nc = UINavigationController(rootViewController: pollBuilderVC)
-        customPresentViewController(presenter, viewController: nc, animated: true, completion: nil)
+        nc.isNavigationBarHidden = true
```

Comment:
I know that you didn't name the variable but can you rename `nc` to something like `pollBuilderNavigationController` or something like that